### DISCUSSION TITLE
【诈骗】加入敲敲群，一起完成委托吧

### DISCUSSION THREAD

**Discussion by yssxrc** [share121/inter-knot#3346] [其他]
###  _**加入敲敲群，一起完成委托吧**_

![qrcode_1724079782737](https://github.com/user-attachments/assets/b5f7bb28-a346-435f-b3d3-9952e900e8f3)

敲敲群号：1040183634
点击链接加入：https://qm.qq.com/q/UmCAXEaOg8
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Comment by yssxrc** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by 1luik** [comment]
已申请
